general/document_splitter.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "W-dlkukAsGuQ"
+   },
+   "source": [
+    "# Overview\n",
```

Comment:
Do we need all this intro to document AI here? We can just put in README in the top directory or something to stay consistent with other notebooks.

Reply:
agree, done